config/dj_config.py

```diff
@@ -95,4 +95,4 @@ def set_configuration(user_name: str, file_name: str = None):
 
 
 if __name__ == "__main__":
-    set_configuration(sys.argv[1])
+    set_configuration(sys.argv[1], sys.argv[2])
```

Comment:
To handle the case for the second optional argument when it is not given I suggest:
```suggestion
    set_configuration(*sys.argv[1:])
```